syz-ci/manager.go

```diff
@@ -271,6 +271,10 @@ func (mgr *Manager) pollAndBuild(lastCommit string, latestInfo *BuildInfo) (
 		mgr.buildFailed = needsUpdate
 		if commit.Hash != lastCommit && needsUpdate {
 			lastCommit = commit.Hash
+			if err := mgr.repo.PushCurrentCommitAsTagTo(mgr.cfg.GitCache); err != nil {
```

Comment:
Do we want to remember all commits we tried to build? Or only the ones that we build successfully and began fuzzing?

Reply:
I don't see big difference. Do you like one option more than other?
Saving everything we decrease the update size but increase frequency.

Reply:
No strong preference, just want to be clear on the intention. For coverage aggregations, we should only need the commits of the successful builds, right?

Reply:
Yes. For coverage we need only successful builds.

Reply:
Let's move to post-build.
PTAL